src/libcore/mem.rs

```diff
@@ -736,3 +736,99 @@ pub fn discriminant<T>(v: &T) -> Discriminant<T> {
     }
 }
 
+
+#[unstable(feature = "manually_drop", issue = "0")]
```

Comment:
Nit: can you move these attributes below documentation? The established ordering convention is "docs, attributes, implementation".